src/rules/index.ts

```diff
@@ -20,4 +21,5 @@ export const rules = {
   "restrict-new-error": restrictNewError,
   "restrict-react-namespace": restrictReactNamespace,
   "restrict-stop-propagation": restrictStopPropagation,
+  "no-floating-promises": noFloatingPromises,
```

Comment:
@denis-sokolov 

`reword`'s return type is `Rule`, not `RuleDefinition`. I will poke at this, but mentioning in case you know the answer already.

![CleanShot 2025-02-13 at 12 39 23@2x](https://github.com/user-attachments/assets/bf5724d0-20dc-4af8-a112-8279aedd7a4f)

Reply:
I think we need to add a `defaultOptions` field to `Rule` in `@denis-sokolov/eslint-plugin`. Its value can be `[]`.

Reply:
Try @denis-sokolov/eslint-plugin 18.1.1.

Reply:
That worked.